project-backend/Controllers/JobController.cs

```diff
@@ -123,6 +125,39 @@ public JobResponseObject AddJob([FromBody] AddJobQueryObject job)
             return response;
         }
 
+        [HttpPut]
+        [Authorize]
+        [Route("Edit")]
+        [ProducesResponseType(typeof(JobResponseObject), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(Error), StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        public IActionResult EditJob([FromBody] EditJobQueryObject editedJob)
+        {
+            var userIdClaim = HttpContext.User.GetUserIdClaim();
+            try
+            {
+                var updatedJob = _jobProvider.EditJob(editedJob.Id, editedJob.Name, editedJob.Description, int.Parse(userIdClaim.Value));
+                var response = new JobResponseObject
+                {
+                    UserId = updatedJob.UserId,
+                    PostDate = updatedJob.PostDate,
+                    Description = updatedJob.Description,
+                    Done = updatedJob.Done,
+                    Id = updatedJob.Id,
+                    Name = updatedJob.Name,
```

Comment:
```suggestion
                    Name = updatedJob.Name,
                    Images = updatedJob.Images.Select(img => img.URL).ToArray(),
```
We should be including the images here, otherwise the response will contain ```"images": null```